### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by benjamin-cates**
Add logging to file functionality and added --debug flag to replace print_verbose

### CODE REVIEW

File: classification/config.py [hunk 1]
```diff
@@ -153,10 +164,10 @@ def get_git_hash(self):
         #becuase of this, this leads to a lot of linting issues ¯\(ツ)/¯
         # pylint: disable=no-member
         except git.exc.InvalidGitRepositoryError: # pyright: ignore [reportGeneralTypeIssues]
-            print("InvalidGitRepositoryError found, this means we cannot save git hash :(")
-            print("You are likely calling a python file outside of this repo") 
-            print("if from command line, cd into acoustic-mutliclass-training")
-            print("then you can run the script again")
+            logger.error("InvalidGitRepositoryError found, this means we cannot save git hash :(")
+            logger.error("You are likely calling a python file outside of this repo") 
+            logger.error("if from command line, cd into acoustic-mutliclass-training")
```
Comment (sprestrelski) [inline classification/config.py:169]: mutli

**Review by sprestrelski — DISMISSED** [review]
works for me - first epoch is off by a little 
image

**Comment by benjamin-cates** [comment]
ok fixed but now your review is stale 😞

File: classification/config.py (hunk 1 above)
Comment (benjamin-cates) [reply classification/config.py:169]: fahgettaboutit

**Comment by Sean1572** [comment]
Are logs still saved on wandb?

**Comment by sprestrelski** [comment]
> Are logs still saved on wandb?

Yes

**Review by sprestrelski — APPROVED** [review]

**Pull request merged**